ansible/roles/elastic-beats/tasks/main.yml

```diff
@@ -89,6 +93,28 @@
                     - json.hostname
                     - json.time
                   ignore_missing: true
+          - type: container
```

Comment:
I'm afraid container approach won't work if we restart, or start a new one due to fail

Reply:
I wasn't able to get Tenderdash to write logs to files, how did we do this last time? With something like `nohup`?

I restarted a few containers during testing to change the logging verbosity and filebeat was still able to push logs to elasticsearch, why do you think this won't work?